src/app/dim-ui/CustomStatTotal.tsx

```diff
@@ -59,7 +59,7 @@ export function StatTotalToggle({
                 .map((statHash) => (
                   <StatToggleButton
                     key={statHash}
-                    stat={defs.Stat.get(statHash)}
+                    stat={statHash && defs.Stat.get(statHash)}
```

Comment:
https://sentry.io/share/issue/5e9935174ac744ffa6a8ebdc3084adc7/

This does not seem to be the right fix, as it is `def` that is somehow undefined

Reply:
this is a known race condition where the settings page can i think load before the manifest is loaded, and therefore the stat icons which rely on the manifest, fail to look up

Reply:
We should just block the search page on loading defs